components/carte-etat/carte-etat-component.jsx

```diff
@@ -140,62 +138,58 @@ class CarteEtat extends PureComponent {
                 </Button>
               </center>
             </div>
-          ) : (
-            [
```

Comment:
The key issue was coming for the presence of an array. If we remove it, we get an linting error because two conditional `?` are nested.